pkg/controller/template/render.go

```diff
@@ -388,7 +388,7 @@ func cloudConfigFlag(cfg RenderConfig) interface{} {
 
 	flag := "--cloud-config=/etc/kubernetes/cloud.conf"
 	switch cfg.Infra.Status.PlatformStatus.Type {
-	case configv1.AWSPlatformType, configv1.AzurePlatformType, configv1.GCPPlatformType, configv1.OpenStackPlatformType, configv1.VSpherePlatformType:
+	case configv1.AWSPlatformType, configv1.AzurePlatformType, configv1.GCPPlatformType, configv1.OpenStackPlatformType, configv1.PowerVSPlatformType, configv1.VSpherePlatformType:
```

Comment:
@Prashanth684 this is not required for powervs platform, this applies only for the in-tree cloud-providers